config/common-config.yaml

```diff
@@ -418,17 +418,13 @@ prometheus:
     requestlimit:
       cpu:
         - name: worker
-          pattern: ".*-worker-.*"
           limit: 80
         - name: elastisys
-          pattern: ".*-elastisys-.*"
           limit: 80
       memory:
         - name: worker
-          pattern: ".*-worker-.*"
           limit: 80
         - name: elastisys
-          pattern: ".*-elastisys-.*"
           limit: 80
```

Comment:
If I understand correctly, this list determines which node groups we will look at. 
I wonder if it would be better to just look at all node groups that are available. The thing I dislike about this is that we would have to remember to add a new entry to this list if we create a new node group. I suspect that it will be rather common for wc workers to be split into several different groups.
But I also see that we might not want these alerts on e.g. AMS nodes or control plane nodes.

What do you think about this? 

Note that I do think your changes fulfill the task very well. So this might be a bit out of scope.

Reply:
Yeah that is correct. I also had the same thoughts about this list, but opted against changing the behavior for the same reason you brought up. Not sure what is preferable :thinking:

Reply:
I'm not following the reason for why we wouldn't want this? Isn't the purpose of the `node-group` label specifically for these alerts, so then we shouldn't have multiple different if we don't want it. We have the `node-type` that we should use for scheduling purposes if that's necessary

And regarding us not wanting AMSes, couldn't we add a "excludeList" here. So that we can specify what node groups we don't want to get alerts for, alternatively we always ignore empty `node-group` and just set it to that if we don't want it.

But I might've misunderstood something completely here

Reply:
I think what I would prefer the most is just, if you want capacity alerts for a node group you label it with `node-group`, since that is the only purpose of the label right now, and if not, don't label them. My hesitation was mostly just changing the behavior from only looking at worker and elastisys nodes, but maybe there wasn't a particular reason for that in the first place. So I'm fine with just making this look at any nodes with the `node-group` label, like the other capacity alerts. Then later down the line, if we find a need for picking and choosing which particular alerts are enabled for which `node-groups`, then maybe we can add some `excludeList`. Thoughts?

Reply:
The label was originally made for the usage alerts, not these request alerts. They are very similar, but different enough that I thought it was worth discussing.

An exclude list could be a good way to work around this if we wanted to. I think that we should also always ignore empty `node-group`, but I think that there might be some situations where we might want to alert on high usage but not high requests (such as the AMS nodes that have very stable requests that we might not need alerts for, but usage alerts would still be good).

Reply:
Right, I was confusing this with the usage alerts and I agree that they are different. And in that case I would probably prefer to, as we said, go with all `node-group` and add an exclude list. I guess the only issue in that case is that we can't have only the resource alert and not the capacity alert. But I don't see any situation where we wouldn't want that.

Reply:
Question: we will now lose the ability to configure the limit for when to alert for each node group, is that also fine? @viktor-f @Xartos

Reply:
For now I think it can be ok to lose the ability to configure limit per group.
I also think that for requests we only need alert per group, not individual nodes.